lang/fr_FR.ini

```diff
@@ -73,178 +73,178 @@ Tag = "Tag"
 Tags = "Tags"
 Title = "Titre"
 Uncategorized = "Sans catégorie"
-Uncategorized_comment = "Sujets qui n'ont pas besoin de catégorie, ou qui ne conviennent a aucun catégorie existante"
+Uncategorized_comment = "Sujets qui n'ont pas besoin de catégorie, ou qui ne conviennent à aucun catégorie existante"
```

Comment:
"Catégorie" grammatical gender is feminine in French (genre féminin).
So I would write "Sujets qui n'ont pas besoin de catégorie, ou qui ne conviennent à aucune catégorie existante"